polaris-react/src/components/LegacyCard/LegacyCard.scss

```diff
@@ -243,3 +255,15 @@
     }
   }
 }
+
+.FirstSectionPadding {
+  #{$se23} & {
+    padding-top: var(--p-space-4);
+  }
+}
+
+.LastSectionPadding {
+  #{$se23} & {
+    padding-bottom: var(--p-space-4);
+  }
+}
```

Comment:
These are the classes to programatically add and remove from first and last elements of the card to give a `16px` padding all around the parent (even though the padding is applied to the children). This is because `LegacyCard`'s without sections have no padding